### ISSUE TITLE
invalid signature of inject

### ISSUE THREAD

**Issue by jacekkarczmarczyk** [vuejs/composition-api#290]
Vue 3.0 https://github.com/vuejs/vue-next/blob/master/packages/runtime-core/src/apiInject.ts#L29):
```ts
export function inject<T>(key: InjectionKey<T> | string): T | undefined
```

Vue 2.x composition api (https://github.com/vuejs/composition-api/blob/master/src/apis/inject.ts#L35):
```ts
export function inject<T>(key: InjectionKey<T> | string): T | void;
```

`void` instead of `undefined` makes impossible to use non-null assertion operator

**Comment by yoyoys** [comment]
I've installed `0.5.0`, still got wrong type.

**Comment by LinusBorg** [comment]
This was merged after 0.5.0.

Well hopefully soon have a fresh release with this type correction and a lot more from #311
